docs/pt/README.md

```diff
@@ -0,0 +1,39 @@
+---
+home: true
+heroImage: /logo.png
+actionText: Começar →
+actionLink: /pt/guia/
+footer: MIT Licensed | Copyright © 2016-present jc21.com
+---
+
+<div class="features">
+  <div class="feature">
+    <h2>Conecte-se</h2>
+    <p>
+      Exponha serviços da Web em sua rede &middot;
+      SSL grátis com Let's Encrypt &middot;
+      Projetado com a segurança em mente &middot;
+      Perfeito para redes domésticas
+    </p>
+  </div>
+  <div class="feature">
+    <h2>Proxy Hosts</h2>
+    <p>Exponha os serviços da Web de sua rede privada e conecte-se em qualquer lugar.</p>
+  </div>
+  <div class="feature">
+    <h2>UI atrativa</h2>
+    <p>Baseado no Tabler, a interface é um prazer de usar. Configurar um servidor nunca foi tão divertido.</p>
```

Comment:
```suggestion
    <p>Baseado no Tabler, é prazeroso usar a interface. Configurar um servidor nunca foi tão divertido assim.</p>
```

Reply:
I can't see any verbal agreement in `a interface é um prazer de usar` phrase, IMHO.
`Configurar um servidor nunca foi tão divertido`, I think we missed completeness in this sentence. It does not express what actually they meant. The idea is to express a fun alternative for configuring server over the others alternatives. That's why I added `assim` in the end of the sentence.

Reply:
That make sense